components/CreateIspoForm.js

```diff
@@ -112,59 +120,47 @@ function SelectTokenField() {
             startAdornment: <InfoIcon fontSize="small" />,
           }}
         >
-          <MenuItem value="dummy">Dummy token</MenuItem>
+          {/* TODO show current token contract name/address */}
+          <MenuItem value="dummy">Set token details</MenuItem>
```

Comment:
This should ideally mirror the `contractName` field value of the form @AndrejZelonka